package.json

```diff
@@ -14,6 +14,7 @@
   },
   "author": "",
   "dependencies": {
+    "@aws-sdk/client-lambda": "3.699.0",
```

Comment:
I think you could move this to devdependencies as the AWS lambda runtime environment has the aws-sdk packages installed by default. Try to move it to devdependencies.